force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +325,14 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+            Decimal total = 0.0; //Variable to store the total value, initiated at zero
+            for (Decimal amount : moneyInWallet) { //captures each value in amount variable for use in the loop.  
+            total += amount; //Add the current amount to the total
+            if (total > 40) { //Check if the total exceeds 40 dollars
```

Comment:
Try adding a variable for the number 40 so it's not a magic number  https://en.wikipedia.org/wiki/Magic_number_(programming)